colorbrewer/README.md

```diff
@@ -0,0 +1,19 @@
+# cljsjs/textures
```

Comment:
The readme still refers other package in this header and in dependency vector below.